### ISSUE TITLE
line_break_between_method_arguments anonymous class

### ISSUE THREAD

**Issue by grachevko** [PedroTroller/PhpCSFixer-Custom-Fixers#17]
Fixer handle open brace of nested anonymouse class as root class

```
===================================================================
--- app/src/Command/LoopedCommand.php	(revision f2167e1e8ce5bacd7c63d29c93cd149b80e526a0)
+++ app/src/Command/LoopedCommand.php	(date 1529951423000)
@@ -94,13 +94,13 @@
             private $isLoop;
 
             public function __construct(SymfonyStyle $io, bool $isLoop)
-            {
+    {
                 $this->io = $io;
                 $this->isLoop = $isLoop;
             }
 
             public function __call($name, $arguments)
-            {
+    {
                 if (!$this->isLoop) {
                     return null;
                 }
```

**Comment by PedroTroller** [comment]
@grachevko thanks. I've reproduced the problem. I'll try to fix it today.

**Comment by PedroTroller** [comment]
@grachevko fixed with v2.10.2.
Thanks a lot :+1:
